src/send/types.ts

```diff
@@ -0,0 +1,364 @@
+/* tslint:disable:interface-name */
+export interface IBrandSnippet {
+  format: "handlebars"; // could support other formats in the future
+  name: string;
+  value: string;
+}
+export interface IBrandSnippets {
+  items: IBrandSnippet[];
+}
+
+export interface IBrandColors {
+  // CSS compliant color value
+  primary?: string;
+  secondary?: string;
+  tertiary?: string;
+}
+
+interface IBrandTemplate {
+  backgroundColor?: string;
+  blocksBackgroundColor?: string;
+  enabled: boolean;
+  footer?: string;
+  head?: string;
+  header?: string;
+  width?: string;
+}
+
+export interface IBrandTemplateOverride extends IBrandTemplate {
+  mjml?: IBrandTemplate;
+  footerBackgroundColor?: string;
+  footerFullWidth?: boolean;
+}
+
+export interface IBrandSettingsSocialPresence {
+  inheritDefault?: boolean;
+  facebook?: {
+    url: string;
+  };
+  instagram?: {
+    url: string;
+  };
+  linkedin?: {
+    url: string;
+  };
+  medium?: {
+    url: string;
+  };
+  twitter?: {
+    url: string;
+  };
+}
+
+
+export interface IBrandSettingsEmail {
+  templateOverride?: IBrandTemplateOverride;
+  head?: {
+    inheritDefault: boolean;
+    content?: string;
+  };
+  footer?: {
+    content?: object;
+    inheritDefault?: boolean;
+    markdown?: string;
+    social?: IBrandSettingsSocialPresence;
+  };
+  header?: {
+    inheritDefault?: boolean;
+    barColor?: string;
+    logo?: {
+      href?: string;
+      image?: string;
+    };
+  };
+}
+
+export interface IBrandSettingsInApp {
+  borderRadius?: string;
+  disableMessageIcon?: boolean;
+  fontFamily?: string;
+  placement?: "top" | "bottom" | "left" | "right";
+  widgetBackground?: {
+    topColor?: string;
+    bottomColor?: string;
+  };
+  colors?: {
+    invertHeader?: boolean;
+    invertButtons?: boolean;
+  };
+  icons?: {
+    bell?: string;
+    message?: string;
+  };
+  preferences?: {
+    templateIds: string[];
+  };
+}
+
+export type IActionButtonStyle = "button" | "link";
+export type IAlignment = "center" | "left" | "right" | "full";
+
+export interface ElementalContent {
+  version: "2022-01-01";
+  brand?: any; // TODO
+  elements: ElementalNode[];
+}
+
+export type ElementalNode =
+  | ElementalTextNode
+  | ElementalMetaNode
+  | ElementalChannelNode
+  | ElementalImageNode
+  | ElementalActionNode
+  | ElementalDividerNode
+  | ElementalGroupNode
+  | ElementalQuoteNode;
+
+export interface ElementalTextNode extends ElementalBaseNode {
+  type: "text";
+  content: string;
+  format?: "markdown";
+  locales?: ElementalLocales;
```

Comment:
See comment below (on the quote node)

```suggestion
  locales?: {
    [locale: string]: {
      content: string;
    };
  };
```